api/anubis/views/public/auth.py

```diff
@@ -14,9 +15,15 @@
 from anubis.lms.courses import get_course_context
 from anubis.lms.submissions import fix_dangling
 from anubis.utils.auth.oauth import OAUTH_REMOTE_APP as provider
+from anubis.utils.auth.oauth import OAUTH_REMOTE_APP_GITHUB as github_provider
 
 auth_ = Blueprint("public-auth", __name__, url_prefix="/public/auth")
 oauth_ = Blueprint("public-oauth", __name__, url_prefix="/public")
```

Comment:
Maybe we should rename this to `nyu_oauth_` to fit with the `github_oauth_` naming structure.

Reply:
Noted